### DISCUSSION TITLE
Prestashop stripe addons problem

### DISCUSSION THREAD

**Discussion by manupuy** [symfony/symfony#59206] [Q&A]
Hello,
I'm just a prestatshop user (no particular computer skills) and when i try to upgrade (or install) my stripe addons (online paiement), i have an error with the message below. One on my friend (developper) advise me to ask help on this chat... Can you help me?

[PrestaShop] Fatal error in module file: /srv/data/web/vhosts/[www.lbvselection.com/htdocs/app/bootstrap.php.cache:3225](http://www.lbvselection.com/htdocs/app/bootstrap.php.cache:3225)
Uncaught Symfony\Component\Console\Exception\LogicException: The command defined in "Symfony\Bundle\FrameworkBundle\Command\WorkflowDumpCommand" cannot have an empty name. in /srv/data/web/vhosts/[www.lbvselection.com/htdocs/vendor/symfony/symfony/src/Symfony/Component/Console/Command/Command.php:68](http://www.lbvselection.com/htdocs/vendor/symfony/symfony/src/Symfony/Component/Console/Command/Command.php:68) Stack trace: #0 [internal function]: Symfony\Component\Console\Command\Command->__construct() #1 /srv/data/web/vhosts/[www.lbvselection.com/htdocs/app/bootstrap.php.cache(3030)](http://www.lbvselection.com/htdocs/app/bootstrap.php.cache(3030)): ReflectionClass->newInstance() #2 /srv/data/web/vhosts/[www.lbvselection.com/htdocs/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Application.php(150)](http://www.lbvselection.com/htdocs/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Application.php(150)): Symfony\Component\HttpKernel\Bundle\Bundle->registerCommands(Object(Symfony\Bundle\FrameworkBundle\Console\Application)) #3 /srv/data/web/vhosts/[www.lbvselection.com/htdocs/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Application.php(131)](http://www.lbvselection.com/htdocs/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Application.php(131)): Symfony\Bundle\FrameworkBundle\Console\Application->registerCommands() #4 /srv/data/web/vhosts/[www.lbvselection.com](http://www.lbvselection.com/)

**Comment by stof** [comment]
which version of `symfony/framework-bundle` and `symfony/console` are you using ?

**Comment by manupuy** [comment]
Hello. I'm aurait i can't tell you. Where can i find this information?

**Comment by manupuy** [comment]
My prestashop version is 1.7.2
